### DISCUSSION TITLE
issue: isTouched always false

### DISCUSSION THREAD

**Discussion by to-sh-1** [react-hook-form/react-hook-form#6744] [Q&A] [answered]
### Version Number

7.12.2

### Codesandbox/Expo snack

https://codesandbox.io/s/react-hook-form-v7-controller-forked-1j5ie?file=/src/index.js

### Steps to reproduce

1. Go to codesandbos
2. Scroll down to NumberFormat Example
3. Type in NumberFormat
4. Check console logs

### Expected behaviour

The NumberFormat registers touched

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by getTobiasNielsen** [comment, marked as answer]
@toshiSat The version in the code example is `7.11.1`

With `7.11.1`, `7.12.2` and the latest `7.17.1` it works as expected.

See example: https://codesandbox.io/s/react-hook-form-v7-controller-forked-892x5?file=/src/index.js

**Comment by bluebill1049** [comment]
<img width="939" alt="Screen Shot 2021-10-08 at 9 05 19 am" src="@getTobiasNielsen ps://user-images.githubusercontent.com/10513364/136468560-f66250b9-94fc-4976-a389-898dee36ebae.png">

thanks @getTobiasNielsen for the help and answer.
